doc/release-notes-23093.md

```diff
@@ -0,0 +1,8 @@
+Notable changes
+===============
+
+Updated RPCs
+------------
+
+- a new RPC `newkeypool` has been added, which will flush (entirely
```

Comment:
why not also: https://github.com/bitcoin/bitcoin/pull/23093/files#diff-55604f94acaa0e77d34f79745e2677fc2c7573d5de48e7f5d5a7a4b554f7a293R7-R9

Reply:
because we have `upgradetohd` for it, which is compatible with mnemonics also